themes/pom.xml

```diff
@@ -16,6 +16,11 @@
         <dir.common>src/main/resources/theme/keycloak/common/resources</dir.common>
         <dir.account2>src/main/resources/theme/keycloak.v2/account/src</dir.account2>
         <args.npm.install>ci --no-optional --ignore-scripts</args.npm.install>
+        <!-- ignore folders for incremental builds by the maven build cache plugin -->
+        <maven.build.cache.exclude.1>${project.basedir}/src/main/resources/theme/keycloak/common/resources/node_modules</maven.build.cache.exclude.1>
+        <maven.build.cache.exclude.2>${project.basedir}/src/main/resources/theme/keycloak/common/resources/web_modules</maven.build.cache.exclude.2>
+        <maven.build.cache.exclude.3>${project.basedir}/src/main/resources/theme/keycloak.v2/account/src/node_modules</maven.build.cache.exclude.3>
+        <maven.build.cache.exclude.4>${project.basedir}/src/main/resources/theme/keycloak.v2/account/src/web_modules</maven.build.cache.exclude.4>
```

Comment:
Could you elaborate on this?

Reply:
When the maven cache plugin calculates the checksum for the cached artifacts, it shouldn't look into the folders with content generated during the build. Otherwise it will mix up the caching as the checksum before the build is different from the one after the build.

Having it in such properties is not as clean as I want it to be, but that's the way they are doing it at the moment.

Reply:
Removed from PR, as this is reduced to only M4 compatibilty.